app/ui-message/client/message.html

```diff
@@ -141,7 +141,7 @@
 							</button>
 						{{/each}}
 					</div>
-					<div class="message-actions__menu" aria-haspopup="true">
+					<div class="message-actions__menu" aria-haspopup="true" title="Options">
```

Comment:
- Please use translation key here
- I think "more" is more suitable for this case

Reply:
I was thinking about translating but couldn't figure out how to translate here. Could you give me some hints? 
I saw {useTranslation} is being used in some files but i wasn't able to use it here

Reply:
I figured it out. Thanks for the review!